api/docker-compose.deploy.yml

```diff
@@ -49,5 +49,7 @@ services:
       - DATABASE_URL=${DATABASE_URL:?"Provide a URL for the alerts database."}
       - KOBO_PW=${KOBO_PW:?"Add a password to access KOBO data."}
       - KOBO_USERNAME=${KOBO_USERNAME:-kobo_user}
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
```

Comment:
If this is a requirement in prod, you can make docker-compose complain if it is not set, see "KOBO_PW" above for an example